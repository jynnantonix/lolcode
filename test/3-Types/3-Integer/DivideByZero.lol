This test checks that dividing by zero gives an error.
HAI
	I HAS A var ITZ -123
	VISIBLE QUOSHUNT OF var AN 0 MKAY?
KTHXBAI
