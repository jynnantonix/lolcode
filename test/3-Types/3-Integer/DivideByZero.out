Unexpected error: division by zero
